iset.mm

```diff
@@ -63850,6 +63854,14 @@ all reals (greatest real number) for which all positive reals are greater.
       JZLZCAMTNMOZEPEQVBVGCARUMUAMVHEUBCAVAEUCUDVCAJZVEVFVIVCMVDVCMJZVICUEZSVDC
       MTVCUFZUGZIZMEVMCUHUIVLMJZVNMUJCMCMVLMUKVOVJTVCUNVKULSUOUPUQABVCURUSUT $.
 
+    $( The left injection restricted to the first class of a disjoint union is
+       an injective function from the first class into the disjoint union.
+       (Contributed by AV, 28-Jun-2022.) $)
```

Comment:
Maybe I missed in in the previous commit, but since B is arbitrary, maybe:
>The left injection restricted to a class is an injective function into the disjoint union of that class (on the left) with any other class.

And same with inrresf1 (and same with the versions without injectivity).

Reply:
Although yor proposal is correct and more precise, I think my description is also correct and a little bit simpler: it should be clear by the theorem itself that B is arbitrary and that A is the left side of the disjoint union. Therefore, I would like to keep the current comment. Maybe I can replace "first/second class of the disjoint union" by "left/right class of the disjoint union".

Reply:
Ok, as you prefer.

Reply:
comments adjusted